src/main/java/org/mule/extension/sftp/internal/connection/SftpClient.java

```diff
@@ -74,6 +77,7 @@
  */
 public class SftpClient {
 
+  private static final String CONFIG_FILE_NAME = "mule_sshd_config";
```

Comment:
This constant has already been defined before. Please define it publicly only once.

Reply:
yeah this is actually not used, will remove